Client/Frontend/Menu/Toolbar/MenuToolbarItem.swift

```diff
@@ -0,0 +1,31 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+
+struct MenuToolbarItem {
+
+    private let iconName: String
+    private let privateModeIconName: String
+
+    let title: String
+
+    var icon: UIImage? {
+        return UIImage(named: iconName)
+    }
+
+    var privateModeIcon: UIImage? {
+        return UIImage(named: privateModeIconName)
+    }
+
+    func iconForMode(isPrivate isPrivate: Bool = false) -> UIImage?  {
+        return isPrivate ? privateModeIcon : icon
```

Comment:
These are temporary until we have created the `AppState`, at which time they will be refined to be less cumbersome.